scripts/algorand_node_log.json

```diff
@@ -0,0 +1,49 @@
+{
+    "algorand_node_log" : {
+        "title" : "Algornad Node Log Format",
+        "description" : "Log format for logrus, used by go-algorand.",
+        "url" : "https://github.com/sirupsen/logrus",
+		"file-pattern": "node.log",
```

Comment:
`file-pattern` is used to avoid potential conflicts when `lnav` chooses the file format to use.
I've replaced it with `samples` which defines some log line samples and uses them for conflict resolution instead (seems like this is the more common approach).